src/frontend/screens/explore/Explore.jsx

```diff
@@ -1,7 +1,30 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { Loader, Posts } from "../../components";
+import { Box, Text } from "@chakra-ui/react";
+import { useDispatch, useSelector } from "react-redux";
+import { getAllPosts } from "../../features";
 
 function Explore() {
-	return <div>Explore Page</div>;
+	const dispatch = useDispatch();
+	const { status, posts } = useSelector((state) => state.posts);
+	useEffect(() => {
+		if (status === "idle") {
+			dispatch(getAllPosts());
+		}
+	}, [dispatch, status]);
+
+	return (
+		<Box p="4" boxShadow="xl">
+			{status === "loading" && <Loader />}
```

Comment:
status=status.success like this constant will be easier and cleaner

Reply:
yes i will add it